### PR TITLE
[WIFI-7697] Move deploy related files from wlan-testing to wlan-cloud-ucentral-deploy

### PR DESCRIPTION
**Pull request by oblom0v**

### CODE REVIEW

File: .github/workflows/ow_k8s.yml [hunk 1]
```diff
@@ -77,10 +77,11 @@ jobs:
     - name: Checkout repo with Helm values
       uses: actions/checkout@v2
       with:
-        path: wlan-testing
+        repository: Telecominfraproject/wlan-cloud-ucentral-deploy
+        path: wlan-cloud-ucentral-deploy
```
Comment (dunaev-opsfleet) [inline .github/workflows/ow_k8s.yml:81]: We should probably also checkout the target branch to get deployment related to the changes for the specific version

File: .github/workflows/uc_loadsim.yml [hunk 2]
```diff
@@ -120,10 +120,11 @@ jobs:
         EOF
 
     # Deploy CloudSDK with OWLS in K8s
-    - name: Checkout Testing repo
+    - name: Checkout wlan-cloud-ucentral-deploy repo
```
Comment (dunaev-opsfleet) [inline .github/workflows/uc_loadsim.yml:123]: Same note regarding checkout to branch

File: .github/workflows/uc_loadsim.yml [hunk 3]
```diff
@@ -372,10 +373,11 @@ jobs:
...
```
Comment (dunaev-opsfleet) [inline .github/workflows/uc_loadsim.yml:376]: Same note regarding checkout to branch

File: .github/workflows/uc_loadsim.yml [hunk 4]
```diff
@@ -543,10 +545,11 @@ jobs:
       - docker-compose-sqlite
       - docker-compose-psql
     steps:
-    - name: Checkout Testing repo
+    - name: Checkout wlan-cloud-ucentral-deploy repo
```
Comment (dunaev-opsfleet) [inline .github/workflows/uc_loadsim.yml:548]: Same note regarding checkout to branch

File: .github/workflows/uc_loadsim.yml [hunk 5]
```diff
@@ -695,10 +698,11 @@ jobs:
       - docker-compose-psql
       - kubernetes-sqlite
     steps:
-    - name: Checkout Testing repo
+    - name: Checkout wlan-cloud-ucentral-deploy repo
```
Comment (dunaev-opsfleet) [inline .github/workflows/uc_loadsim.yml:701]: Same note regarding checkout to branch

File: .github/workflows/ucentralgw-dev-deployment.yaml [hunk 6]
```diff
@@ -91,10 +91,11 @@ jobs:
     - name: Checkout repo with Helm values
       uses: actions/checkout@v2
       with:
-        path: wlan-testing
+        repository: Telecominfraproject/wlan-cloud-ucentral-deploy
+        path: wlan-cloud-ucentral-deploy
```
Comment (dunaev-opsfleet) [inline .github/workflows/ucentralgw-dev-deployment.yaml:95]: Same note regarding checkout to branch

File: .github/workflows/ucentralgw-qa-deployment.yaml [hunk 7]
```diff
@@ -72,10 +72,11 @@ jobs:
     - name: Checkout repo with Helm values
       uses: actions/checkout@v2
       with:
-        path: wlan-testing
+        repository: Telecominfraproject/wlan-cloud-ucentral-deploy
+        path: wlan-cloud-ucentral-deploy
```
Comment (dunaev-opsfleet) [inline .github/workflows/ucentralgw-qa-deployment.yaml:76]: Same note regarding checkout to branch

File: .github/workflows/ow_k8s.yml (hunk 1 above)
Comment (oblom0v) [reply .github/workflows/ow_k8s.yml:81]: @dunaev-opsfleet yeah missed that, thanks!

**Review by dunaev-opsfleet — APPROVED** [review]

**Comment by dunaev-opsfleet** [comment]
@oblom0v seems good, but I've just understood that those changes will require additional changes in Quali scenarios since they rely on deployment scripts being in wlan-testing repository

**Comment by oblom0v** [comment]
@dunaev-opsfleet yes, I've created a task for the required changes (WIFI-7707). I won't delete the deployment files from wlan-testing until these changes are done on Quali side.

**Pull request merged**